worker/utils/mongo.js

```diff
@@ -28,6 +31,41 @@ module.exports = {
     return null;
   },
 
+  getMetaCollection() {
+    if (client) {
+      // hardcoded for now because if I use env variables
+      // I'll need to create a meta collection in both databases: `pool` and `pool_test`
+      return client.db('pool').collection('meta');
```

Comment:
Go ahead and create the collection in the staging database.

Reply:
done